cmd/outline-ss-server/config_example.yml

```diff
@@ -20,6 +26,12 @@ services:
         address: "[::]:9000"
       - type: udp
         address: "[::]:9000"
+      - type: websocket-stream
+        web_server: my_web_server
+        path: "/tcp"
```

Comment:
We need a way to prevent probing. We can put this under a `/SECRET/` path, or require authentication. If we require authentication, we need a way to configure that.

The example should illustrate the best practice.